packages/host/app/components/operator-mode/detail-panel.gts

```diff
@@ -367,6 +367,13 @@ export default class DetailPanel extends Component<Signature> {
                 }}
               />
             </div>
+          {{else}}
+            <div
+              class='file-incompatible-message'
+              data-test-detail-panel-file-incompatibility-message
+            >
+              Inspector cannot be used with this file type
```

Comment:
Can we include a message that explains which file types *are* supported?

Reply:
https://github.com/cardstack/boxel/pull/835/commits/86b4d1e1b09b31ff46adc03f63ed8fc1fa0cae92